components/book-detail.styles.js

```diff
@@ -0,0 +1,190 @@
+import { css } from 'lit-element';
+
+export default css`
+
+    :host {
```

Comment:
Try to always use the same indentation (use 2 spaces preferably)

Reply:
Corrected.